### PR TITLE
Add slider to change the marker size

### PR DESCRIPTION
**Pull request by c-bata**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

None

## What does this implement/fix? Explain your changes.

marker-size-change

### CODE REVIEW

**Comment by c-bata** [comment]
@HideakiImamura Could you review this PR?

**Comment by HideakiImamura** [comment]
It seemed to me that we could focus on smaller areas. This is because it seems to me that we may want to reduce the size of a dot, but not to make it larger than it currently is.

**Comment by c-bata** [comment]
Thanks for your input. I'm thinking of narrowing down the range to 1-10. What do you think?

**Comment by HideakiImamura** [comment]
Sounds good. Could you apply the change?

File: optuna_dashboard/ts/components/GraphHistory.tsx [hunk 1]
```diff
@@ -140,6 +144,20 @@ export const GraphHistory: FC<{
             />
           </RadioGroup>
         </FormControl>
+        <FormControl>
+          <FormLabel component="legend">Marker size:</FormLabel>
+          <Slider
+            defaultValue={5}
+            marks={true}
+            min={1}
+            max={20}
```
Comment (c-bata) [inline optuna_dashboard/ts/components/GraphHistory.tsx:153]:
```suggestion
            max={10}
```

File: optuna_dashboard/ts/components/GraphHistory.tsx [hunk 2]
```diff
@@ -263,6 +283,20 @@ export const GraphHistoryMultiStudies: FC<{
...
```
Comment (c-bata) [inline optuna_dashboard/ts/components/GraphHistory.tsx:292]:
```suggestion
            max={10}
```

**Comment by c-bata** [comment]
Applied. PTAL.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Pull request merged**